src/reynir_correct/settings.py

```diff
@@ -720,9 +882,56 @@ def _handle_morphemes(s: str) -> None:
                     )
         Morphemes.add(m, boundlist, freelist)
 
+    @staticmethod
+    def _handle_ritmyndir(s: str) -> None:
+        """Handle data from Ritmyndir in Stórasnið in BÍN/DIM"""
+        split = s.strip().split(";")
+        if len(split) != 13:
+            raise ConfigError(
+                "Expected lemma, id, cat, wrong_word_form, correct_word_form, tag, eink, malsnid, stafs, aslatt, beyg, age, ref"
+            )
+        ref = split[12].strip()
+        if "SAGA" in ref or ref in {"MILTON", "HALLGP-4", "KLIM", "ONP"}:
+            # Skipping errors from very old references, don't represent errors in Modern Icelandic
+            # NOTE: Causes many common errors to be skipped, since they are first found there
```

Comment:
Is that correct? It seems like the code is keeping the newest (last) entry for each word form, throwing previous ones away. And these sources (MILTON...) are never added to the dict.

Reply:
Don't remember exactly what went down but seems to be a remnant from older code where entries weren't added if the wrong form was already there. Deleted the comment.